tests/integration/test_integration.py

```diff
@@ -87,7 +88,7 @@ def update_output(value):
             len('hello world')
         )
 
-        assert_clean_console(self)
+        self.assertTrue(self.is_console_clean())
```

Comment:
Seems like a kind of funny pattern, and can't report what is in the console if it fails. Do we ever test that the console is *not* clean? Perhaps better as just `self.assert_clean_console()` or something like that?

Reply:
personally, I am not a big fan of the assertTrue, assertEqual pattern. I like the pytest way, just using the native `assert`.  this one can be 
```python
assert self.is_console_clean(), "the browser console should contain no js error"
```
 I will clean up these in a separate PR